it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionControllerForOverseasISpec.scala

```diff
@@ -94,8 +99,39 @@ class SubscriptionControllerForOverseasISpec extends BaseISpec with OverseasDesS
           deleteKnownFact = 1,
           createKnownFact = 1,
           enrol = 1,
+          amls = 1,
           complete = 1)
       }
+
+      "creating amls record fails with 409 Conflict because a record already exists" in {
```

Comment:
This is grouped in the "return a successful response containing the ARN" grouping atm. Could just move it out of that level, or create another grouping for something like "return conflict response" which includes the other conflict response tests.